Fix some minor problems

michael-hartmann opened the pull request:
This PR fixes some minor issues:

- Remove some unused variables.
- Include header file stdlib.h in files that use the exit-function.
- Define the macro Pi explicitly.

The first two points are pretty straight-forward: Those changes prevent warning when compiling the code.

The last point is more subtile: The macro M_PI is not defined in standard C, so compiling fails for example if you use gcc with the flag -std=c99.

michael-hartmann committed:
Include header file <stdlib.h>

In both files the function exit is used which is defined in stdlib.h.

michael-hartmann committed:
Remove unused variables

michael-hartmann committed:
Remove unused variables

michael-hartmann committed:
Don't rely that M_PI is defined

M_PI is not defined in standard C. For example, compiling this code using gcc
with -std=c99 aborts, because M_PI is undefined. So, to be standard-complaint,
we shouldn't rely on M_PI.

tadeu commented on include/cquadpack.h:
```diff
@@ -9,7 +9,7 @@
 #define LIMIT     500
 #define MAXP1     21
-#define Pi      M_PI
+#define Pi      3.141592653589793
```
Thanks for the fixes. Could you please use a few more digits, just in case the compiler pre-optimizes some multiplications using 80-bits floating point precision? Thanks!

(`3.14159265358979323846` would be fine)

tadeu commented:
@michael-hartmann Would you like to be a co-maintainer of this package? This will give you read/write/merge privileges.

michael-hartmann committed:
use M_PI if possible, otherwise use our constant

Though M_PI is not part of any C standard, the constant is defined by many
compilers. If it is defined, use M_PI. If it is not used, use a representation
with about 22 correct digits.

michael-hartmann commented:
I changed the part about Pi a bit.

If M_PI is defined, Pi is just M_PI. So the original behaviour is not unchanged. If M_PI is not defined, use an explicit representation of Pi with about 22 correct digits.

Also, I would be happy to be co-maintainer. :)

tarcisiofischer commented on include/cquadpack.h:
```diff
@@ -9,7 +9,11 @@
+#ifdef M_PI
+#define Pi      M_PI
+#else
+#define Pi      3.114159265358979323846
```
Note that Pi is being defined as 3.1141...

michael-hartmann committed:
fix value of pi

michael-hartmann commented:
Oh, I'm sorry, I fixed that.

tadeu commented:
@michael-hartmann Added as a maintainer, but feel free to continue creating PRs if you want the code to be peer reviewed (seems to be a healthy practice for the project 😃)
